orx-mesh-generators/src/demo/kotlin/DemoExtrude03.kt

```diff
@@ -0,0 +1,102 @@
+import org.openrndr.WindowMultisample
+import org.openrndr.application
+import org.openrndr.color.ColorRGBa
+import org.openrndr.draw.DrawPrimitive
+import org.openrndr.draw.shadeStyle
+import org.openrndr.extra.camera.Orbital
+import org.openrndr.extra.color.presets.CORAL
+import org.openrndr.extra.color.presets.NAVY
+import org.openrndr.extra.color.presets.TEAL
+import org.openrndr.extra.meshgenerators.buildTriangleMesh
+import org.openrndr.extra.meshgenerators.extrudeContourAdaptive
+import org.openrndr.extra.meshgenerators.extrudeContourSteps
+import org.openrndr.math.Polar
+import org.openrndr.math.Vector3
+import org.openrndr.math.asDegrees
+import org.openrndr.math.asRadians
+import org.openrndr.shape.Circle
+import org.openrndr.shape.Path3D
+import kotlin.math.PI
+import kotlin.math.exp
+
+fun main() {
+    application {
+        configure {
+            width = 800
+            height = 800
+            multisample = WindowMultisample.SampleCount(8)
+        }
+        program {
+            fun spiralPath(a: Double, k: Double, cycles: Double, steps: Int, direction:Double = 1.0): Path3D {
+                val points = (0 until steps).map {
+
+                    val theta = ((PI * 2.0 * cycles) / steps) * it
+                    val radius = a * exp(k * theta)
+
+                    val c = Polar(theta.asDegrees, radius).cartesian
+                    c.xy0
+                }
+                return Path3D.fromPoints(points, false)
+            }
+
+            val spiral = buildTriangleMesh {
+                for (i in -1 .. 1) {
+                    if (i == 0) continue
```

Comment:
I think this could be rewritten as an IntProgression along the lines of `for (i in -1..1 step 2)`.